src/source/codegen.js

```diff
@@ -8,4 +8,6 @@
 const codegen = require('shift-codegen').default;
 const startJSONStream = require('./start-json-stream');
 
-startJSONStream(ast => codegen(ast));
+startJSONStream({
+	transform: ast => codegen(ast)
```

Comment:
`transform` is perhaps not sufficiently clear as a name.

Reply:
It mirrors the Rust side of naming. You commented on that one in the Rust PR too, but we didn't come up with something better for a generic "transform arbitrary value to arbitrary value" API.

Reply:
Can you just comment quickly?

Maybe something as simple as:
```
// Apply `codegen` to the jsonstream.
```

Reply:
I'm usually wary of putting comments that don't bring new information not described by the code, but sure. In this case it felt like code is already readable as almost pure English sentence "start JSON stream that transforms AST using codegen".